### DISCUSSION TITLE
Create a RouterLink alias to Link

### DISCUSSION THREAD

**Discussion by prabirshrestha** [TanStack/router#658] [Ideas]
Is is common to use router with other UI Libraries. It will be good if `RouterLink` is provided as an alias for `Link` so we don't need to rename it. This plays well with editors that auto imports. Right now I need to keep doing this manually.

```javascript
import { Link } from '@adobe/react-spectrum';
import { Link as RouterLink } from '@tanstack/router';

<Link>
 <RouterLink to="/">Home</RouterLink>
</Link>
```

**Comment by morellodev** [comment]
I don't think it would add value to the library – as you said, you can always alias the imports manually. If you find this solution tedious, you could add a file `tanstack-router.ts` that imports and exports things from the original lib with the names you prefer:

```ts
export { Link as RouterLink, ... } from "@tanstack/router";
```

And then, in your files:

```ts
import { RouterLink, ... } from "<path/to/tanstack-router.ts>";
